### PR TITLE
github action to build and publish docker image

### PR DESCRIPTION
**Pull request by btkostner**

### CODE REVIEW

**Comment by benwaffle** [comment]
shouldn't we just publish on merge to master?

**Comment by btkostner** [comment]
@benwaffle We should on merge to master, and daily to ensure the docs are up to date.

File: .github/workflows/publish.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+name: Publish Docker Image
+
+on:
+  schedule:
+    - cron:  '0 0 * * *'
```
Comment (benwaffle) [inline .github/workflows/publish.yml]: Should we run it during daylight hours EST or PST so that people can fix it quicker should it break?

Comment (btkostner) [reply .github/workflows/publish.yml]: Good question. I think that's smart.

**Pull request merged**